package.json

```diff
@@ -10,19 +10,18 @@
     "build": "yarn build:prod && yarn build:dev",
     "build:dev": "cross-env BUILD=development rollup -c",
     "build:prod": "cross-env BUILD=production rollup -c",
-    "cleanup": "yarn cleanup:coverage && yarn cleanup:dist",
-    "cleanup:coverage": "rimraf coverage",
+    "cleanup": "yarn cleanup:dist",
     "cleanup:dist": "rimraf dist",
     "dev-server": "webpack-dev-server --config webpack.config.dev.js --progress --colors",
-    "test": "mocha test/**/*.js",
-    "cover": "yarn cleanup:coverage && istanbul cover _mocha test",
+    "test": "jest",
+    "test:watch": "jest --watchAll",
+    "cover": "yarn test --coverage",
```

Comment:
This won't work with the version of yarn that travis comes with by default. I will try to force latest stable in the future but for now let's just do `yarn test -- --coverage`